client/devdocs/async-load/placeholder.jsx

```diff
@@ -0,0 +1,37 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import PostItem from 'blocks/post-item';
+import SitePlaceholder from 'blocks/site/placeholder';
+
+export default class DevdocsAsyncLoadPlaceholder extends React.PureComponent {
+	static displayName = 'DevdocsAsyncLoadPlaceholder';
+	static propTypes = {
+		count: PropTypes.number.isRequired,
+	};
+
+	render() {
+		return (
+			<Main className="design async-load__devdocs-placeholder">
+				{ Array( this.props.count )
+					.fill( 1 )
+					.map( () => (
+						<div>
+							<SitePlaceholder />
+							<PostItem />
+						</div>
+					) ) }
```

Comment:
Check out `lodash.range()` for doing stuff like this. The `fill-map` is redundant, both because we could have filled with the JSX and avoided the map but also because we can do maybe more common patterns like…

```js
range( count ).map( () => <div><SitePlaceHolder /><PostItem /></div> )
```

Reply:
There's `lodash.times` too, which avoids creating the intermediary map.

Reply:
Ah, cool. I've not used `lodash` in awhile, sorry I am not super familiar with the API 😅

Reply:
That said, is there any benefit to using `times( count, func)` over `range( count ).map( func )`? Is it just style? The latter seems more ES6-y and friendly to my eyes as an outsider. I can change it easily if it's just a style preference, but I don't see any win to using `times` over `range`.

Reply:
no strong preference :smile: